src/main/java/datawave/data/normalizer/AbstractGeometryNormalizer.java

```diff
@@ -49,7 +50,15 @@
      */
     @Override
     public String normalize(String geoString) throws IllegalArgumentException {
-        return normalizeDelegateType(createDatawaveGeometry((G) parseGeometry(geoString)));
+        try {
+            return normalizeDelegateType(createDatawaveGeometry((G) parseGeometry(geoString)));
+        } catch (Exception e) {
+            if (validHash(geoString)) {
+                return geoString;
+            }
+            Throwables.propagateIfPossible(e, IllegalArgumentException.class);
+        }
+        throw new IllegalArgumentException("Unable to normalize geo string " + geoString);
```

Comment:
Does it make sense to check for a valid hash first, and only attempt to normalize if an invalid hash is found? That would save all the work spent to build a stack trace